README.md

```diff
@@ -61,8 +68,8 @@ data-commons/
 1. **Clone the repository:**
 
    ```bash
-   git clone https://github.com/HxLu03/DataCommons.git
-   cd data-commons
+   git clone https://github.com/alexandra5279/REDMANE_react.js.git
```

Comment:
Can you please change this to git clone git@github.com:WEHI-RCPStudentInternship/REDMANE_react.js.git